Library/Homebrew/cask/cmd/automerge.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require "cask/cmd/abstract_internal_command"
+require "tap"
+require "utils/formatter"
+require "utils/github"
+
+module Cask
+  class Cmd
+    class Automerge < AbstractInternalCommand
+      OFFICIAL_CASK_TAPS = [
+        "homebrew/cask",
+        "homebrew/cask-drivers",
+        "homebrew/cask-eid",
+        "homebrew/cask-fonts",
+        "homebrew/cask-versions",
+      ].freeze
+
+      def run
+        raise "This command may only be run by Homebrew maintainers." unless ENV["HOMEBREW_DEVELOPER"]
```

Comment:
It shouldn’t stop merging this now, but we should consider for the future (as an alternative to this line) adding a `has_write_permission` method to `utils/github` to check if the caller can write to the repo, and if not nip this in the bud early (“You do not have permissions to merge pull requests in this repo”) instead of checking for `HOMEBREW_DEVELOPER` (which users can set as well) and then try to merge and fail.

Reply:
Done.